test/comparisons.jl

```diff
@@ -96,7 +98,8 @@ const INTERVAL_TYPES = [Interval, AnchoredInterval{Ending}, AnchoredInterval{Beg
         @test !issubset(later, earlier)
 
         @test isempty(intersect(earlier, later))
-        # @test union(a, b) == Interval(1, 5, false, true)
+        @test_throws ArgumentError merge(earlier, later)
+        @test union([earlier, later]) == [earlier, later]
```

Comment:
These should be combined into a single interval as there is nothing that can exist between these two intervals.